scripts/makebumpver

```diff
@@ -330,120 +324,65 @@ class MakeBumpVer:
-
-                        if valid:
-                            rhbz.add("%s: rhbz#%s" % (action, ckbug))
-
-                            # Remove the summary bug's Resolves action if it is for the same bug
-                            if action != 'Resolves':
-                                summary_str = "Resolves: rhbz#%s" % summary_bug
-                                if summary_bug and ckbug == summary_bug and summary_str in rhbz:
-                                    rhbz.remove(summary_str)
-                        else:
-                            bad = True
-
-                        if self.skip_all:
-                            print("*** Bug %s Related commit %s is allowed\n" % (bug, commit))
-                            continue
-
-                        not_correct = not self._isRHELBugInCorrectState(ckbug, commit, summary)
-                        not_acked = not self._isRHELBugAcked(ckbug, commit, summary)
-
-                        if valid and action == 'Resolves' and (not_correct or not_acked):
-                            bad = True
-                        elif valid and action == 'Related':
-                            # A related bug needs to have acks, and if it is the same as the summary
-                            # It overrides the summary having different fixed-in or state
-                            if self._isRHELBugAcked(ckbug, commit, summary):
-                                print("*** Bug %s Related commit %s is allowed\n" % (bug, commit))
-                                if ckbug == summary_bug:
-                                    bad = False
-                            else:
-                                bad = True
-
-                if len(rhbz) == 0 and not self.skip_all:
-                    print("*** No bugs referenced in commit %s\n" % commit)
-                    bad = True
-
-                rpm_log.append((summary.strip(), list(rhbz)))
+                bad, bugs = self._check_rhel_issues(commit, summary, body, author)
+                rpm_log.append(("%s (%s)" % (summary.strip(), author), list(bugs)))
             else:
                 rpm_log.append(("%s (%s)" % (summary.strip(), author), None))
 
-            if bad:
-                bad_bump = True
-
-        if bad_bump:
+        if bad:
             sys.exit(1)
 
         return rpm_log
 
-    def _writeNewConfigure(self, newVersion):
+    def _check_rhel_issues(self, commit, summary, body, author):
+        issues = set()
+        bad = False
+
+        summary = summary + f" ({author})"
+
+        for bodyline in body:
+            m = re.match(r"^(Resolves|Related):\ +jira#\w+-\d+.*$", bodyline)
+            if not m:
+                continue
+
+            actionre = re.search("(Resolves|Related)", bodyline)
+            bugre = re.search(r"\w+-\d+", bodyline)
+
+            if actionre and bugre:
+                action = actionre.group()
+                bug = bugre.group()
```

Comment:
I'd use a single regex to validate this, since then the "parsing" is all contained and linked, but never mind.

<details>
<summary>code</summary>

```
>>> f = re.search(r"^(Resolves|Related):\ +jira#(\w+-\d+).*$", "Resolves: jira#RHELPLAN-1568")
>>> f
<re.Match object; span=(0, 28), match='Resolves: jira#RHELPLAN-1568'>
>>> f.group(0)
'Resolves: jira#RHELPLAN-1568'
>>> f.group(1)
'Resolves'
>>> f.group(2)
'RHELPLAN-1568'
>>> f = re.findall(r"^(Resolves|Related):\ +jira#(\w+-\d+).*$", "Resolves: jira#RHELPLAN-1568")
>>> f
[('Resolves', 'RHELPLAN-1568')]
```
</details>